src/pages/ServiceGraph/CytoscapeConfig.ts

```diff
@@ -33,9 +33,10 @@ export class CytoscapeConfig {
         css: {
           width: 2,
           content: 'data(text)',
-          'target-arrow-shape': 'triangle',
```

Comment:
Strange so they changed the symbol name for target-arrow-shape somewhere along the way ;

Reply:
Hmmm not really...
Check this image: https://issues.jboss.org/secure/attachment/12431890/screenshot-1.png
I used ´vee´ because I like it a little more than ´triangle´. But I can change it back to triangle if you think it's better.

Reply:
Nope. It looks good!